cmd/timestamp-cli/app/verify.go

```diff
@@ -67,7 +75,7 @@ func runVerify() (interface{}, error) {
 		return nil, fmt.Errorf("error reading request from file: %w", err)
 	}
 
-	certChainPEM := viper.GetString("cert-chain")
+	certChainPEM := viper.GetString("certificate-chain")
```

Comment:
This and the CertPool parameter to VerifyTimestampResponse can be removed now because VerifyOpts contains the roots/intermediates